sdk/python/jobs/pipelines/1e_pipeline_with_registered_components/pipeline_with_registered_component_in_registry.ipynb

```diff
@@ -0,0 +1,249 @@
+    "    # Check if given credential can get token successfully.\n",
+    "    credential.get_token(\"https://management.azure.com/.default\")\n",
+    "except Exception as ex:\n",
+    "    # Fall back to InteractiveBrowserCredential in case DefaultAzureCredential not work\n",
+    "    credential = InteractiveBrowserCredential()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fa8c5101",
+   "metadata": {},
+   "source": [
+    "## 1.3 Get a handle to the workspace and the registry"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3c246768",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get a handle to workspace\n",
+    "ml_client_ws = MLClient(\n",
+    "    credential, \"<SUBSCRIPTION_ID>\", \"<RESOURCE_GROUP>\", \"<AML_WORKSPACE_NAME>\"\n",
+    ")\n",
+    "ml_client_registry = MLClient(credential, registry_name=\"azureml\")\n",
+    "\n",
+    "# Retrieve an already attached Azure Machine Learning Compute.\n",
+    "cluster_name = \"cpu-cluster-D13-v2\"\n",
+    "\n",
+    "# Note above compute has an MSI attached to it, which is necessary to get credentials and run cli/sdk v2 on that compute"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85c2a79d",
+   "metadata": {},
+   "source": [
+    "# 2. Load registered components to create pipeline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "475e505c",
+   "metadata": {},
+   "source": [
+    "## 2.1 Use worskpace handle to load the components from workspace"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c0d0d321",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_downloader = ml_client_registry.components.get(\n",
+    "    name=\"download_model\", label=\"latest\"\n",
+    ")\n",
+    "mlflow_converter = ml_client_registry.components.get(\n",
+    "    name=\"mlflow_converter\", label=\"latest\"\n",
+    ")\n",
+    "model_registration = ml_client_registry.components.get(\n",
+    "    name=\"register_model\", label=\"latest\"\n",
+    ")\n",
+    "model_deployment = ml_client_registry.components.get(\n",
+    "    name=\"deploy_model\", label=\"latest\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6723387f",
+   "metadata": {},
+   "source": [
+    "# 3. Build pipeline using above components"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f68c3e15",
+   "metadata": {},
+   "source": [
+    "## 3.1 Connect components and build pipeline framework"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8a5d0dc8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@pipeline(\n",
+    "    description=\"Model Publishing Pipeline\",\n",
+    ")\n",
+    "def model_publishing_pipeline():\n",
+    "\n",
+    "    model_downloading_node = model_downloader(\n",
+    "        model_uri_type=\"git\",\n",
+    "        model_uri=\"https://huggingface.co/ai4bharat/indic-bert\",\n",
```

Comment:
>"https://huggingface.co/ai4bharat/indic-bert [](http://example.com/codeflow?start=24&length=44)

Could we please bert-base-uncased instead?

Reply:
Done